### ISSUE TITLE
AttributeError: 'NoneType' object has no attribute 'capture'

### ISSUE THREAD

**Issue by ghost** [SerpentAI/D3DShot#5]
Hello!

I'm trying to use the basic examples, but when trying to create a screenshot with:

>>> import d3dshot
>>> d = d3dshot.create()
>>> d.screenshot()

I'm getting 

File "<stdin>", line 1, in <module>
  File "C:\Users\Administrator\AppData\Local\Programs\Python\Python37\lib\site-packages\d3dshot\d3dshot.py", line 89, in screenshot
    frame = self.display.capture(self.capture_output.process, region=region)
AttributeError: 'NoneType' object has no attribute 'capture'

What am I doing wrong?

**Comment by nbrochu** [comment]
You aren't doing anything wrong but for some reason, no primary display seem to be detected on your system. D3DShot's behavior is to detect all displays and set `self.display` to the primary one. 

Can you check what the output of `d.displays` is? If it's an empty list, I'm going to need more information to tackle this. If it's not empty, can you:

1. Paste the output here.
2. Try `d.display = d.displays[0]` and then `d.screenshot()`

Is there anything non-standard about your setup? I can't come up with anything when trying to think of ways that Windows could not have any primary displays. And just to confirm, we are talking about Windows 10 64 bit?

**Comment by WowSuchRicky** [comment]
I have this issue too. Output of d.display is:
```
[<Display name=Unknown adapter=NVIDIA GeForce GTX 1080 Ti resolution=3440x1440 rotation=0 scale_factor=1.0 primary=False>, <Display name=Unknown adapter=NVIDIA GeForce GTX 1080 Ti resolution=3840x2160 rotation=0 scale_factor=1.5 primary=False>]
```

I believe I fixed it in #21 !

**Comment by nbrochu** [comment]
Yes #21 fixes it. Thanks @Ricky54326
